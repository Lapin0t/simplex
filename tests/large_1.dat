45
36
0 0 0 0 0 0 25 0 -183 0 0 195 0 -55 0 0 0 7 -74 -124 79 0 0 0 0 -87 0 0 0 -75 90 0 0 0 152 0 0 0 0 0 0 0 0 0 0
4976 -38998 -23015 -14672 -14617 -33997 12794 -17460 -27642 -13695 -14070 17887 -46825 8467 9216 5962 4416 3833 -8394 11671 -2320 15207 -21546 -11136 -9281 9110 -7828 -796 -6398 -18099 -51591 12273 4369 -11798 -10850 -38477
93 177 0 -45 34 116 -124 73 0 -183 0 0 -95 0 0 -57 0 0 0 0 32 0 0 30 -61 -45 57 0 0 0 0 -148 0 0 0 0 119 0 0 -85 0 0 0 -3 0
0 -40 0 0 0 85 0 0 0 0 -11 0 0 39 0 149 -191 0 0 0 0 0 0 0 0 0 -73 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 76
1 0 -54 -136 -163 -143 0 179 0 0 -174 0 0 0 0 0 -136 0 -192 0 0 145 0 0 0 0 145 0 122 3 0 -127 23 0 0 0 0 0 0 0 103 0 0 0 0
2 0 -47 163 0 0 0 0 0 0 0 0 0 0 166 0 -141 0 0 107 0 0 -181 0 36 0 0 0 0 -92 0 22 0 0 -85 0 0 0 0 0 0 0 -135 0 0
-125 0 -6 -109 0 0 0 0 0 0 0 0 33 0 33 0 0 0 0 -117 19 0 -85 163 178 0 0 -5 0 184 -165 49 0 14 185 0 0 0 0 196 0 143 0 -26 0
-75 0 -49 0 0 0 0 0 177 0 0 0 34 0 36 0 0 0 -108 -197 -20 0 147 0 0 0 0 56 0 91 0 0 189 0 0 0 0 -89 140 -175 100 0 0 -67 119
0 0 0 0 -110 100 0 0 0 0 0 0 0 0 0 0 0 0 -18 0 0 0 -36 0 0 0 0 0 115 0 0 0 0 139 0 0 0 0 -72 0 0 -158 150 0 0
0 0 162 0 0 0 12 0 0 0 0 52 0 62 80 0 -70 0 0 0 0 0 60 0 -60 0 -194 0 0 0 0 -8 0 -17 0 0 0 0 0 -132 0 0 0 10 0
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 -159 0 9 -144 0 0 59 0 0 0 -160 0 0 0 0 0 131 -48 0 -9 0 159 0 0 0 -16
139 -59 0 0 0 0 0 0 0 138 0 55 0 0 0 114 0 0 0 0 136 0 0 0 -111 -4 0 0 0 86 -18 146 31 0 0 0 73 0 176 0 0 -87 -156 -62 129
0 0 0 0 14 130 0 -80 0 0 0 0 0 0 0 0 19 0 0 0 -102 63 0 0 0 0 0 -102 0 -94 0 0 82 0 0 0 140 0 0 0 0 156 85 123 -29
53 0 0 0 0 0 0 0 0 148 0 0 5 19 0 0 104 0 0 0 0 0 0 0 -82 0 0 0 0 -190 0 -154 0 0 0 0 0 -149 -134 0 0 -40 17 -166 115
0 -35 0 0 0 0 123 -164 -3 0 0 52 -70 0 0 0 -169 0 0 0 0 0 0 0 0 0 0 0 42 138 141 126 0 0 0 3 104 91 192 0 0 0 0 0 0
0 -110 0 0 0 -164 0 148 0 0 0 122 0 0 0 0 0 0 0 0 -125 85 0 0 0 0 0 0 0 0 0 0 0 0 -90 56 0 -136 -43 0 0 0 0 0 0
84 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 -179 0 0 0 96 0 -84 0 169 0 195 0 52 0 120 0 0 -31 0 0 -74 0
0 0 0 89 40 0 192 0 0 0 0 -93 0 136 0 159 0 0 -46 0 0 0 0 0 20 0 0 76 0 0 0 137 67 0 -30 -88 0 0 0 0 104 0 0 0 0
0 0 0 -88 -157 17 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 -35 -132 0 0 0 -10 0 -36 0 -175 0 0 0 176 0 0 0 0 0 -191 0 52 0 0
0 0 0 0 0 0 0 59 0 0 -17 0 0 98 -101 0 0 0 0 8 0 50 0 0 0 0 0 -14 0 0 0 0 0 0 38 0 0 0 24 156 84 0 0 0 -115
-158 0 0 0 0 0 0 0 -18 56 0 0 0 199 0 0 0 0 0 -82 0 0 0 0 0 0 158 0 0 0 0 -168 0 0 0 0 85 3 -155 0 0 -181 0 0 0
0 0 0 0 0 0 129 36 0 0 -196 136 0 0 70 0 0 -192 111 0 59 0 0 0 79 -140 24 0 0 0 0 0 0 0 -2 74 0 0 -18 0 0 40 0 0 0
0 0 0 177 0 0 0 10 35 -73 0 0 0 0 0 -28 0 0 0 0 167 78 -200 109 -32 56 0 0 0 0 0 -34 0 0 88 0 142 0 0 0 -149 0 0 0 -4
0 33 0 0 0 0 0 0 0 0 0 0 0 -80 119 0 0 0 0 83 39 78 0 0 0 0 173 0 0 0 77 -96 0 152 0 196 0 0 0 0 70 -50 -5 0 -8
-103 0 0 -122 0 0 0 0 185 0 0 91 0 0 0 0 -123 -48 0 -3 0 143 0 0 0 0 0 57 0 0 0 0 -41 0 0 -100 0 -60 0 0 0 0 0 0 175
55 0 0 0 -50 -143 0 -132 0 0 0 0 0 0 0 0 0 52 0 0 0 0 0 0 0 -14 195 30 0 0 0 0 0 0 -125 0 0 0 0 0 0 22 -74 -124 0
76 0 42 -69 0 -177 105 0 0 103 0 0 0 0 0 0 0 0 -161 0 0 140 155 0 0 0 0 0 -164 86 -108 0 0 0 0 17 0 0 8 140 0 0 0 0 -159
163 0 0 175 0 0 0 0 0 0 0 0 0 100 0 0 0 0 -132 36 0 0 0 -67 0 28 0 0 -18 0 0 0 0 -137 -5 0 0 0 0 0 -61 0 170 -164 0
0 0 0 0 0 13 0 0 0 114 39 0 0 0 0 152 0 18 0 0 0 0 0 -200 0 118 0 -81 -123 0 0 190 23 169 109 0 54 0 -26 0 -11 0 0 0 0
0 0 103 0 0 144 0 0 43 0 0 0 0 167 55 0 0 0 0 0 0 0 0 -58 0 0 -75 -3 -26 0 0 0 28 0 -143 123 0 0 0 18 50 0 0 -160 160
0 0 25 0 -126 0 0 -185 0 0 0 0 0 -45 148 0 0 0 -77 -44 0 0 0 0 150 0 0 0 -101 -177 0 0 0 0 0 0 0 0 0 151 0 0 170 52 0
0 0 0 0 -181 -188 0 -138 0 22 0 13 0 0 0 -19 0 0 0 0 0 0 0 0 109 0 0 -167 0 -189 0 -94 0 0 0 -93 0 0 0 0 -83 0 0 0 98
0 0 0 0 0 -196 0 -113 0 -81 0 63 0 0 0 0 -186 0 -141 0 0 0 0 0 0 -196 -171 0 0 0 0 0 137 0 0 0 0 0 0 0 0 0 0 0 0
-14 133 21 0 0 0 -143 0 20 0 0 69 -80 0 0 0 0 0 0 0 0 0 0 0 158 0 0 0 11 124 0 0 -27 0 0 0 -136 -13 0 0 0 0 0 0 0
0 -137 0 -200 0 0 0 122 0 0 0 0 0 0 0 81 -79 0 0 78 0 0 -192 0 0 0 64 0 -14 0 0 0 0 -83 0 20 0 0 0 0 0 0 0 0 142
54 -164 -42 0 0 0 167 0 -86 0 0 0 0 0 -56 0 0 0 94 0 0 0 0 131 0 0 64 -159 0 0 29 0 0 55 0 180 0 0 0 0 0 -72 0 133 0
-166 -183 -90 0 -19 0 -100 0 0 -178 0 0 104 0 0 0 0 0 0 0 0 0 0 0 -119 0 0 0 0 0 0 151 0 0 87 194 0 0 0 0 0 0 0 0 0
0 -191 25 0 -34 0 42 -59 0 0 0 0 0 0 0 15 0 0 0 0 96 0 151 0 -189 0 -45 -194 -122 0 0 114 0 0 0 158 0 0 0 0 0 12 0 0 82
