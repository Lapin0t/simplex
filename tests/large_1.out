INFO: maximize:
INFO:                                                    +25x_6         -183x_8                  +195x_11           -55x_13                              +7x_17  -74x_18 -124x_19  +79x_20                                      -87x_25                             -75x_29  +90x_30                            +152x_34                                                                                          
INFO: subject to:
INFO:    +93x_0 +177x_1          -45x_3  +34x_4 +116x_5 -124x_6  +73x_7         -183x_9                    -95x_12                    -57x_15                                      +32x_20                    +30x_23  -61x_24  -45x_25  +57x_26                                     -148x_31                                     +119x_36                    -85x_39                              -3x_43          <= 4976
INFO:            -40x_1                          +85x_5                                  -11x_10                    +39x_13          +149x_15 -191x_16                                                                                   -73x_26                                                                                                                                                           +76x_44 <= -38998
INFO:      +x_0          -54x_2 -136x_3 -163x_4 -143x_5         +179x_7                 -174x_10                                              -136x_16          -192x_18                   +145x_21                                     +145x_26          +122x_28   +3x_29          -127x_31  +23x_32                                                                +103x_40                                     <= -23015
INFO:     +2x_0          -47x_2 +163x_3                                                                                     +166x_14          -141x_16                   +107x_19                   -181x_22           +36x_24                                      -92x_29           +22x_31                    -85x_34                                                                -135x_42                   <= -14672
INFO:   -125x_0           -6x_2 -109x_3                                                                    +33x_12           +33x_14                                     -117x_19  +19x_20           -85x_22 +163x_23 +178x_24                     -5x_27          +184x_29 -165x_30  +49x_31           +14x_33 +185x_34                                     +196x_39          +143x_41           -26x_43          <= -14617
INFO:    -75x_0          -49x_2                                         +177x_8                            +34x_12           +36x_14                            -108x_18 -197x_19  -20x_20          +147x_22                                      +56x_27           +91x_29                   +189x_32                                      -89x_37 +140x_38 -175x_39 +100x_40                    -67x_43 +119x_44 <= -33997
INFO:                                   -110x_4 +100x_5                                                                                                          -18x_18                             -36x_22                                              +115x_28                                     +139x_33                                      -72x_38                   -158x_41 +150x_42                   <= 12794
INFO:                   +162x_2                          +12x_6                                   +52x_11           +62x_13  +80x_14           -70x_16                                               +60x_22           -60x_24          -194x_26                                       -8x_31           -17x_33                                              -132x_39                             +10x_43          <= -17460
INFO:                                                                                                                                                                    -159x_19            +9x_21 -144x_22                    +59x_25                            -160x_29                                              +131x_35  -48x_36            -9x_38          +159x_40                             -16x_44 <= -27642
INFO:   +139x_0  -59x_1                                                         +138x_9           +55x_11                            +114x_15                                     +136x_20                            -111x_24   -4x_25                             +86x_29  -18x_30 +146x_31  +31x_32                             +73x_36          +176x_38                    -87x_41 -156x_42  -62x_43 +129x_44 <= -13695
INFO:                                    +14x_4 +130x_5          -80x_7                                                                        +19x_16                            -102x_20  +63x_21                                              -102x_27           -94x_29                    +82x_32                            +140x_36                                     +156x_41  +85x_42 +123x_43  -29x_44 <= -14070
INFO:    +53x_0                                                                 +148x_9                     +5x_12  +19x_13                   +104x_16                                                                 -82x_24                                     -190x_29          -154x_31                                              -149x_37 -134x_38                    -40x_41  +17x_42 -166x_43 +115x_44 <= 17887
INFO:            -35x_1                                 +123x_6 -164x_7   -3x_8                   +52x_11  -70x_12                            -169x_16                                                                                                     +42x_28 +138x_29 +141x_30 +126x_31                              +3x_35 +104x_36  +91x_37 +192x_38                                                       <= -46825
INFO:           -110x_1                         -164x_5         +148x_7                          +122x_11                                                                         -125x_20  +85x_21                                                                                                              -90x_34  +56x_35          -136x_37  -43x_38                                                       <= 8467
INFO:    +84x_0                                                                                                                                                                                              -179x_23                             +96x_27           -84x_29          +169x_31          +195x_33           +52x_35          +120x_37                    -31x_40                    -74x_43          <= 9216
INFO:                            +89x_3  +40x_4         +192x_6                                   -93x_11          +136x_13          +159x_15                    -46x_18                                               +20x_24                    +76x_27                            +137x_31  +67x_32           -30x_34  -88x_35                                     +104x_40                                     <= 5962
INFO:                            -88x_3 -157x_4  +17x_5                                                                                                                                     -35x_21 -132x_22                             -10x_26           -36x_28          -175x_30                            +176x_34                                              -191x_40           +52x_42                   <= 4416
INFO:                                                            +59x_7                  -17x_10                    +98x_13 -101x_14                                       +8x_19           +50x_21                                               -14x_27                                                        +38x_34                             +24x_38 +156x_39  +84x_40                            -115x_44 <= 3833
INFO:   -158x_0                                                          -18x_8  +56x_9                            +199x_13                                               -82x_19                                                       +158x_26                                     -168x_31                                      +85x_36   +3x_37 -155x_38                   -181x_41                            <= -8394
INFO:                                                   +129x_6  +36x_7                 -196x_10 +136x_11                    +70x_14                   -192x_17 +111x_18           +59x_20                             +79x_24 -140x_25  +24x_26                                                                  -2x_34  +74x_35                    -18x_38                    +40x_41                            <= 11671
INFO:                           +177x_3                          +10x_7  +35x_8  -73x_9                                               -28x_15                                     +167x_20  +78x_21 -200x_22 +109x_23  -32x_24  +56x_25                                               -34x_31                    +88x_34          +142x_36                            -149x_40                              -4x_44 <= -2320
INFO:            +33x_1                                                                                             -80x_13 +119x_14                                      +83x_19  +39x_20  +78x_21                                     +173x_26                             +77x_30  -96x_31          +152x_33          +196x_35                                      +70x_40  -50x_41   -5x_42            -8x_44 <= 15207
INFO:   -103x_0                 -122x_3                                 +185x_8                   +91x_11                                     -123x_16  -48x_17            -3x_19          +143x_21                                               +57x_27                                      -41x_32                   -100x_35           -60x_37                                                       +175x_44 <= -21546
INFO:    +55x_0                          -50x_4 -143x_5         -132x_7                                                                                 +52x_17                                                                 -14x_25 +195x_26  +30x_27                                                       -125x_34                                                        +22x_41  -74x_42 -124x_43          <= -11136
INFO:    +76x_0          +42x_2  -69x_3         -177x_5 +105x_6                 +103x_9                                                                         -161x_18                   +140x_21 +155x_22                                              -164x_28  +86x_29 -108x_30                                      +17x_35                     +8x_38 +140x_39                                     -159x_44 <= -9281
INFO:   +163x_0                 +175x_3                                                                            +100x_13                                     -132x_18  +36x_19                             -67x_23           +28x_25                    -18x_28                                     -137x_33   -5x_34                                               -61x_40          +170x_42 -164x_43          <= 9110
INFO:                                            +13x_5                         +114x_9  +39x_10                                     +152x_15           +18x_17                                              -200x_23          +118x_25           -81x_27 -123x_28                   +190x_31  +23x_32 +169x_33 +109x_34           +54x_36           -26x_38           -11x_40                                     <= -7828
INFO:                   +103x_2                 +144x_5                  +43x_8                                    +167x_13  +55x_14                                                                          -58x_23                    -75x_26   -3x_27  -26x_28                             +28x_32          -143x_34 +123x_35                             +18x_39  +50x_40                   -160x_43 +160x_44 <= -796
INFO:                    +25x_2         -126x_4                 -185x_7                                             -45x_13 +148x_14                             -77x_18  -44x_19                                     +150x_24                            -101x_28 -177x_29                                                                                  +151x_39                   +170x_42  +52x_43          <= -6398
INFO:                                   -181x_4 -188x_5         -138x_7          +22x_9           +13x_11                             -19x_15                                                                         +109x_24                   -167x_27          -189x_29           -94x_31                             -93x_35                                      -83x_40                             +98x_44 <= -18099
INFO:                                           -196x_5         -113x_7          -81x_9           +63x_11                                     -186x_16          -141x_18                                                       -196x_25 -171x_26                                              +137x_32                                                                                                             <= -51591
INFO:    -14x_0 +133x_1  +21x_2                         -143x_6          +20x_8                   +69x_11  -80x_12                                                                                                    +158x_24                             +11x_28 +124x_29                    -27x_32                            -136x_36  -13x_37                                                                <= 12273
INFO:           -137x_1         -200x_3                         +122x_7                                                               +81x_15  -79x_16                    +78x_19                   -192x_22                             +64x_26           -14x_28                                      -83x_33           +20x_35                                                                         +142x_44 <= 4369
INFO:    +54x_0 -164x_1  -42x_2                         +167x_6          -86x_8                                              -56x_14                             +94x_18                                     +131x_23                    +64x_26 -159x_27                    +29x_30                    +55x_33          +180x_35                                               -72x_41          +133x_43          <= -11798
INFO:   -166x_0 -183x_1  -90x_2          -19x_4         -100x_6                 -178x_9                   +104x_12                                                                                                    -119x_24                                                       +151x_31                    +87x_34 +194x_35                                                                                  <= -10850
INFO:           -191x_1  +25x_2          -34x_4          +42x_6  -59x_7                                                               +15x_15                                      +96x_20          +151x_22          -189x_24           -45x_26 -194x_27 -122x_28                   +114x_31                            +158x_35                                               +12x_41                    +82x_44 <= -38477
INFO:   x_0, x_1, x_2, x_3, x_4, x_5, x_6, x_7, x_8, x_9, x_10, x_11, x_12, x_13, x_14, x_15, x_16, x_17, x_18, x_19, x_20, x_21, x_22, x_23, x_24, x_25, x_26, x_27, x_28, x_29, x_30, x_31, x_32, x_33, x_34, x_35, x_36, x_37, x_38, x_39, x_40, x_41, x_42, x_43, x_44, x_45, x_46, x_47, x_48, x_49, x_50, x_51, x_52, x_53, x_54, x_55, x_56, x_57, x_58, x_59, x_60, x_61, x_62, x_63, x_64, x_65, x_66, x_67, x_68, x_69, x_70, x_71, x_72, x_73, x_74, x_75, x_76, x_77, x_78, x_79, x_80 >= 0
INFO: number of variables: 45
INFO: number of constraints: 36
INFO: PHASE 1 (23 bad constraints)
INFO: PHASE 2 (initial base: [39 11 19 16  7 70 72 43 53 47 12 48 51 23 69 80 61 75 42 29 46 18  8 74 22
  1 63 68 28 71 73 25 77 27 57  9])
INFO: number of pivot steps (both phases): 112
INFO: rule used: greedy
BOUNDED
OPTIMAL VALUE:  1105407238182785126847784512096594590804/90213735002325225035345026817505795
SOLUTION:  x_0 = 5307022809994127696142952149715020353/36085494000930090014138010727002318, x_1 = 17205076782934005411527401750416293683/30071245000775075011781675605835265, x_2 = 0, x_3 = 5573757353816930033713160063722915714/18042747000465045007069005363501159, x_4 = 0, x_5 = 0, x_6 = 0, x_7 = 74999625727933242299411558927655510413/360854940009300900141380107270023180, x_8 = 0, x_9 = 0, x_10 = 569497860356065008775313974792581348049/360854940009300900141380107270023180, x_11 = 11873454981831019607052232586027339639/30071245000775075011781675605835265, x_12 = 20074229623990893154088631510881215376/18042747000465045007069005363501159, x_13 = 6765931874802781572757419068961963743/90213735002325225035345026817505795, x_14 = 0, x_15 = 0, x_16 = 291845535808183846842645828339080464519/360854940009300900141380107270023180, x_17 = 28374620428865958220558395958659486154/30071245000775075011781675605835265, x_18 = 29250139947007578567833578818561420284/30071245000775075011781675605835265, x_19 = 0, x_20 = 5433245141206189400648287096864493692/90213735002325225035345026817505795, x_21 = 0, x_22 = 157539576203469373117390481632089055511/360854940009300900141380107270023180, x_23 = 10030927730512686962046386058088989001/60142490001550150023563351211670530, x_24 = 0, x_25 = 0, x_26 = 0, x_27 = 15742381695719477703480591567065083321/24056996000620060009425340484668212, x_28 = 0, x_29 = 0, x_30 = 0, x_31 = 0, x_32 = 0, x_33 = 0, x_34 = 0, x_35 = 0, x_36 = 0, x_37 = 0, x_38 = 0, x_39 = 4675190739960957785235244551255631001/36085494000930090014138010727002318, x_40 = 0, x_41 = 40886877058513988668212434763212092/90213735002325225035345026817505795, x_42 = 0, x_43 = 58685014831237712904645738591649681373/120284980003100300047126702423341060, x_44 = 0
